docs/guides/css-files.md

```diff
@@ -0,0 +1,170 @@
+---
+title: CSS Files
+---
+
+# CSS Files
+
+There are two main ways to manage CSS files in Remix:
+
+- [CSS bundling](#css-bundling)
+- [CSS URL imports](#css-url-imports)
+
+This guide covers the pros and cons of each approach, and provides some recommendations based on your project's specific needs.
+
+## CSS bundling
+
+CSS bundling is the most common approach for managing CSS files in the React community. In this model, styles are treated as module side effects and are bundled into one or more CSS files at the discretion of the bundler. It's simpler to use, requires less boilerplate, and gives the bundler more power to optimize the output.
+
+For example, let's say you have a basic `Button` component with some styles attached to it:
+
+```css filename=components/Button.css
+.Button__root {
+  background: blue;
+  color: white;
+}
+```
+
+```jsx filename=components/Button.jsx
+import "./Button.css";
+
+export function Button(props) {
+  return <button {...props} className="Button__root" />;
+}
+```
+
+To use this component, you can simply import it and use it in your route file:
+
+```jsx filename=routes/hello.jsx
+import { Button } from "../components/Button";
+
+export default function HelloRoute() {
+  return <Button>Hello!</Button>;
+}
+```
+
+When consuming this component, you don't have to worry about managing individual CSS files. CSS is treated as private implementation detail of the component. This is a common pattern in many component libraries and design systems and scales quite nicely.
+
+#### CSS bundling is required for some CSS solutions
```

Comment:
Just wanted to check if you're using h4 to avoid having too many header links on the right side nav

If so [I've got a PR to indent them](https://github.com/remix-run/remix-website/pull/231)

Reply:
Yep, but also to de-emphasise them a bit visually. I'll merge as is but feel free to tweak them if you like.